api/src/pcapi/admin/custom_views/cine_office_pivot_view.py

```diff
@@ -0,0 +1,109 @@
+import typing
+
+from flask import flash
+from flask_admin.form import SecureForm
+from werkzeug.exceptions import Forbidden
+from wtforms import Form
+from wtforms import IntegerField
+from wtforms import StringField
+from wtforms.validators import DataRequired
+
+from pcapi.admin.base_configuration import BaseAdminView
+import pcapi.core.offerers.models as offerers_models
+import pcapi.core.providers.models as providers_models
+from pcapi.core.providers.repository import get_provider_by_local_class
+from pcapi.models import db
+
+
+class CineOfficePivotForm(SecureForm):
+    venue_id = IntegerField("Identifiant lieu", [DataRequired()])
+    account_id = StringField("Compte", [DataRequired()])
+    cinema_id = StringField("Identifiant cinéma", [DataRequired()])
+    api_token = StringField("Clé API", [DataRequired()])
```

Comment:
Pour dissiper les doutes, je préciserais l'origine de l'info:
```suggestion
    venue_id = IntegerField("Identifiant lieu (pass Culture)", [DataRequired()])
    account_id = StringField("Nom de compte (CDS)", [DataRequired()])
    cinema_id = StringField("Identifiant cinéma (CDS)", [DataRequired()])
    api_token = StringField("Clé API (CDS)", [DataRequired()])
```